ios/src/TGMapViewController.mm

```diff
@@ -546,10 +549,11 @@ - (void)respondToShoveGesture:(UIPanGestureRecognizer *)shoveRecognizer
     // don't trigger shove on single touch gesture
     if ([shoveRecognizer numberOfTouches] == 2) {
         if (self.gestureDelegate && [self.gestureDelegate respondsToSelector:@selector(recognizer:didRecognizeShoveGesture:)]) {
-            [self.gestureDelegate mapView:self recognizer:shoveRecognizer didRecognizeShoveGesture:displacement];
-        } else {
-            self.map->handleShoveGesture(displacement.y);
+            if (![self.gestureDelegate mapView:self recognizer:shoveRecognizer didRecognizeShoveGesture:displacement]) {
```

Comment:
Gotcha. So that would change how this one particular gesture delegate works compared to the others, and I think might make it a little confusing: in some instances returning `true` to the delegate function, tangram wouldn't do processing, whereas in other cases it would. 

Maybe a better solution is to allow the number of touches to be customized? I think it would be preferable to have a unified delegate callback system wherein there code only sends the delegate callbacks when tangram also could process it internally. Thoughts?